metaflow/plugins/argo/argo_workflows.py

```diff
@@ -867,11 +873,35 @@ def _container_templates(self):
                 if v is not None and k not in env_vars_to_skip
             }
 
+            # Tmpfs variables
+            use_tmpfs = resources["use_tmpfs"]
```

Comment:
we can move these checks from here and in kubernetes_job.py into the kubernetes decorator.